src/antares/craft/model/settings/study_settings.py

```diff
@@ -9,64 +9,23 @@
-from antares.craft.model.settings.thematic_trimming import (
-    DefaultThematicTrimmingParameters,
-    ThematicTrimmingParametersLocal,
-)
-from antares.craft.model.settings.time_series import DefaultTimeSeriesParameters, TimeSeriesParametersLocal
-from antares.craft.tools.all_optional_meta import all_optional_model
-from antares.craft.tools.ini_tool import get_ini_fields_for_ini
-from pydantic import BaseModel, model_serializer
-
-
-class DefaultStudySettings(BaseModel):
-    general_parameters: DefaultGeneralParameters = DefaultGeneralParameters()
-    # These parameters are listed under the [variables selection] section in the .ini file.
-    # They are required if thematic-trimming is set to true.
-    # https://antares-simulator.readthedocs.io/en/latest/user-guide/solver/04-parameters/#variables-selection-parameters
-    time_series_parameters: DefaultTimeSeriesParameters = DefaultTimeSeriesParameters()
-    # These parameters are listed under the [general] section in the .ini file.
-    # https://antares-simulator.readthedocs.io/en/latest/user-guide/ts-generator/04-parameters/
-    optimization_parameters: DefaultOptimizationParameters = DefaultOptimizationParameters()
-    adequacy_patch_parameters: DefaultAdequacyPatchParameters = DefaultAdequacyPatchParameters()
-    advanced_parameters: DefaultAdvancedParameters = DefaultAdvancedParameters()
-    playlist_parameters: Optional[PlaylistParameters] = None
-    thematic_trimming_parameters: Optional[DefaultThematicTrimmingParameters] = None
-
-
-@all_optional_model
-class StudySettings(DefaultStudySettings):
-    pass
-
-
-class StudySettingsLocal(DefaultStudySettings):
-    general_parameters: GeneralParametersLocal = GeneralParametersLocal()
-    time_series_parameters: TimeSeriesParametersLocal = TimeSeriesParametersLocal()
-    optimization_parameters: OptimizationParametersLocal = OptimizationParametersLocal()
-    adequacy_patch_parameters: AdequacyPatchParametersLocal = AdequacyPatchParametersLocal()
-    advanced_parameters: AdvancedParametersLocal = AdvancedParametersLocal()
-    thematic_trimming_parameters: Optional[ThematicTrimmingParametersLocal] = None
-
-    @model_serializer
-    def serialize(self) -> dict:
-        output_dict = get_ini_fields_for_ini(self)
-        return self._sort_fields_last(output_dict)
-
-    @staticmethod
-    def _sort_fields_last(output_dict: dict) -> dict:
-        new_general = {key: value for key, value in output_dict["general"].items() if key != "readonly"} | {
-            "readonly": output_dict["general"]["readonly"]
-        }
-        new_output = {key: value for key, value in output_dict["output"].items() if key != "result-format"} | {
-            "result-format": output_dict["output"]["result-format"]
-        }
-        output_dict["general"] = new_general
-        output_dict["output"] = new_output
-        return output_dict
+from antares.craft.model.settings.thematic_trimming import ThematicTrimmingParameters
+
+
+@dataclass
```

Comment:
We could make it frozen so that the user does not think that he can update the study simply by setting the properties on this object.